microsoft-edge/progressive-web-apps-chromium/how-to/digital-goods-api.md

```diff
@@ -32,53 +32,111 @@ See:
 
-See [Payment Request API](https://developer.mozilla.org/docs/Web/API/Payment_Request_API) at MDN.
+See:
+* [Payment Request API](https://developer.mozilla.org/docs/Web/API/Payment_Request_API) at MDN.
+
+
+<!-- ====================================================================== -->
+## Enable the Digital Goods API
+
+The Digital Goods API is currently available in pre-release versions of Microsoft Edge.  To use the API, do any of the following:
+* [Use a supported preview channel of Microsoft Edge on your dev machine](#use-a-supported-preview-channel-of-microsoft-edge-on-your-dev-machine).
+* [Start Beta or Stable Edge with command-line arguments on your dev machine](#start-beta-or-stable-edge-with-command-line-arguments-on-your-dev-machine).
+* [Register for the origin trial, then use your token at your website](#register-for-the-origin-trial-then-use-your-token-at-your-website).
+
+Details are below.
+
+
+<!-- ------------------------------ -->
+#### Use a supported preview channel of Microsoft Edge on your dev machine
+
+Approach 1: Run Edge Dev or Edge Canary version 115.0.1860.0 or later.  With this approach, you use the Digital Goods API on your developer machine only, for testing.
+
+That version of Edge has the `msWebAppDigitalGoods`, `AppStoreBilling` and `DigitalGoodsApi` flags enabled, as part of a running experiment.  These flags will be enabled by default when the Digital Goods API reaches Edge Stable release.
+
+To download a preview (insider) channel of Microsoft Edge, see [Become a Microsoft Edge Insider](https://aka.ms/microsoftedge).
+
+
+<!-- ------------------------------ -->
+#### Start Beta or Stable Edge with command-line arguments on your dev machine
+
+Approach 2: Start Edge Stable or Edge Beta with command-line arguments.  With this approach, you use the Digital Goods API on your developer machine only, for testing.
+
+Start Edge Stable or Edge Beta (prior to version 115.0.1860.0) at the command line, using the following command-line argument to enable the set of three flags:
+
+`--enable-features=msWebAppDigitalGoods, AppStoreBilling, DigitalGoodsApi`
+
+
+<!-- ------------------------------ -->
+#### Register for the origin trial, then use your token at your website
+
+Approach 3: Use an origin trial token.  With this approach, you and your users can use Edge Stable, to test the Digital Goods API in production.  You register in the origin trial for the Digital Goods API, and then use your origin trial token at your website.
+
+See: 
+* [Use origin trials in Microsoft Edge](../../origin-trials/index.md)
+* [Digital Goods API](https://developer.microsoft.com/microsoft-edge/origin-trials/trials/4b4a9ead-d912-4349-87b3-25e5e50b4f13) at _Microsoft Edge Developer_ > _Origin Trials_.
 
 
 <!-- ====================================================================== -->
 ## Checking whether the Digital Goods API is available
 
```

Comment:
Add Ref url for window object > getDigitalGoodsService method

Is the present article or article section the draft of the Ref docs for `window` object > `getDigitalGoodsService` method?
@runyuany

Reply:
```
* [getDigitalGoodsService() method](https://wicg.github.io/digital-goods/#getdigitalgoodsservice-method) 
on the `Window` interface, in _Digital Goods API: Draft Community Group Report_.
```
https://wicg.github.io/digital-goods/#getdigitalgoodsservice-method